docs/design/architecture.md

```diff
@@ -10,11 +10,13 @@ Sharing resources with Liqo relies on three different phases:
 
 1. **Discovery**: The cluster looks for available clusters where offload new resources (e.g. neighborhood, dns, manual 
 insertion) and exchange credentials with each other to start communicate.
-2. **Advertisement management**: Clusters shares updates about the resourcing they are willing to export and their
+2. **Advertisement protocol**: Clusters shares updates about the resourcing they are willing to export and their
  capabilities (i.e. Advertisements)
 3. **Resource Sharing**: When a cluster is interested in resources proposed by a certain advertisement, it accepts the
  advertisement. This triggers the establishment of network interconnections and the spawning of a new virtual-kubelet.
 
+[Here](../images/complete-workflow.png) you can find a graphic description of the complete workflow  
```

Comment:
I added this picture of  the complete workflow. It's not very appealing, but I think it helps to understand the global behaviour. Tell me if it's ok, remove or modify it

Reply:
I think that picture is ok and very helpful